cmd/pool/sidecar.go

```diff
@@ -31,31 +33,21 @@ var sidecarOfferCommand = cli.Command{
 	Name:      "offer",
 	Aliases:   []string{"o"},
 	Usage:     "offer a sidecar channel",
-	ArgsUsage: "capacity self_chan_balance lease_duration_blocks",
+	ArgsUsage: "[<full bid args> --auto] | capacity self_chan_balance lease_duration_blocks",
 	Description: `
-	Creates an offer for providing a sidecar channel to another node.`,
-	Flags: []cli.Flag{
-		cli.Uint64Flag{
-			Name: "capacity",
-			Usage: "the total channel capacity of the sidecar " +
-				"channel to offer",
+	Creates an offer for providing a sidecar channel to another node.
+	If the auto flag is specified, then all bid information needs to be 
+	specified as normal. If the auto flag isn't specified, then only 
+	capacity, self_chan_balance and lease_duration_blocks needs to set.`,
+	Flags: append(
+		baseBidFlags,
+		cli.BoolFlag{
+			Name: "auto",
+			Usage: "if true, then the full bid information needs to " +
+				"be specified as automated negotiation will be " +
+				"attempted",
 		},
-		cli.Uint64Flag{
-			Name: "self_chan_balance",
-			Usage: "the number of satoshis that should be pushed " +
-				"to the recipient of the sidecar channel as " +
-				"initial outbound channel balance; amount " +
-				"will be deducted from account that submits " +
-				"bid order, reimbursement must happen out of " +
-				"band, not part of the sidecar protocol",
-		},
-		cli.Uint64Flag{
-			Name: "lease_duration_blocks",
-			Usage: "the number of blocks the resulting leased " +
-				"channel should be open for",
-			Value: uint64(order.LegacyLeaseDurationBucket),
-		},
-	},
+	),
```

Comment:
`sharedFlags` needs to be added here as well, otherwise it'll complain about the `max batch fee rate must be at least 1 sat/vByte`.